packages/plowshare.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Plowshare < Package
+  description 'Command-line tool and engine for managing sharing websites'
+  homepage 'https://github.com/mcrapet/plowshare'
+  version '2.1.7'
+  source_url 'https://github.com/mcrapet/plowshare/archive/v2.1.7.tar.gz'
+  source_sha256 'c17d0cc1b3323f72b2c1a5b183a9fcef04e8bfc53c9679a4e1523642310d22ad'
+
+  depends_on 'recode'
+
+  def self.build
+    system "sed -i 's,PREFIX ?= /usr,PREFIX ?= #{CREW_PREFIX},' Makefile"
```

Comment:
You can `make PREFIX=#{CREW_PREFIX} DESTDIR=${CREW_DEST_DIR} install` without modifying Makefile.